config/routes.rb

```diff
@@ -11,6 +11,12 @@
   namespace :admin do
     get "/users", to: "users#index"
     get "/orders/:order_id", to: "orders#update"
+    get "/merchants/:merchant_id", to: "merchants#show"
+  end
+
+  namespace :merchant do
+    get '/orders/:order_id', to: "orders#show"
+    get '/items', to: 'items#index'
```

Comment:
Just a note that I have more coming here that might conflict. We'll see what happens :)

Reply:
I also may have a conflict here? I think it should be ok...since I'm specifying `only` but don't know if the `resources` syntax will conflict with the normal style.

![Screen Shot 2020-11-01 at 10 13 50 PM](https://user-images.githubusercontent.com/62676321/97829398-890a8080-1c8f-11eb-97cd-70b568c28e24.png)

Reply:
I also may have a conflict here? I think it should be ok...since I'm specifying only but don't know if the resources syntax will conflict with the normal style.

![Screen Shot 2020-11-01 at 10 16 58 PM](https://user-images.githubusercontent.com/62676321/97829524-f7e7d980-1c8f-11eb-8cd8-1dbab3e72dfc.png)